AUTHORS

```diff
@@ -1,4 +1,4 @@
-The following authors have created the source code of "hermione-faildump"
+The following authors have created the source code of "testplane-faildump"
```

Comment:
`faildump`? looks like it was copied with mistake a long time ago. we should fix plugin name here